docs/source/kedro_project_setup/dependencies.md

```diff
@@ -59,7 +36,36 @@ This installs Kedro and dependencies related to the data type group. An example
 To limit installation to dependencies specific to a data type:
 
 ```bash
-pip install "kedro-datasets[<group>.<dataset>]"
+pip install "kedro-datasets[<group>-<dataset>]"
+```
+
+For example, your workflow might require use of the `pandas.ExcelDataset`, so to install its dependencies, run `pip install "kedro-datasets[pandas-exceldataset]"`.
+
+```{note}
+From `kedro-datasets` version 3.0.0 onwards, the names of the optional dataset-level dependencies have been normalised to follow [PEP 685](https://peps.python.org/pep-0685/). The '.' character has been replaced with a '-' character and the names are in lowercase. For example, if you had `kedro-datasets[pandas.ExcelDataset]` in your requirements file, it would have to be changed to `kedro-datasets[pandas-exceldataset]`.
+```
```

Comment:
💯

Reply:
Checking with my understanding, the normalisation is happened automatically, so even if I do `pip install kedro-datasets[pandas-excelDATASETs]` it should still work?